scripts/check-wildcard-deps.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+
+should_error=false
+
+echo "Checking public packages for dependencies with '*' version..."
+
+# Find all package.json files in the current directory and its subdirectories
+while IFS= read -r file; do
+    # Check if "private" is not set to true in the package.json file
+    if ! grep -q '"private":\s*true' "$file"; then
```

Comment:
super nit, but could jq this if you wanted

Reply:
This script came directly from chatgpt 🙈 

Good call though, would be nice to be consistent but I think I'm gonna leave it as is for now